README.md

```diff
@@ -1,16 +1,38 @@
 # dev-ops
-Install Ngninx and GH actions
+Scripts and configuration for app deployment
 
-## Access to your Digital Ocean droplet
+## Option 1 - [Digital Ocean App Platform](https://docs.digitalocean.com/products/app-platform/)
+Automate the deployment of your app using the App Platform without having to manually configure any servers or sites.\
+Works by first scanning your GitHub repo and looking for common build/run commands (customisable), then waiting for a push against the `main` branch before executing those commands to automatically build and deploy your app\
+Extra info [here](https://docs.digitalocean.com/products/app-platform/languages-frameworks/nodejs/). Example ExpressJS app [here](https://github.com/digitalocean/sample-expressjs)
+
+### Instructions
+1. Create the app on DO [here](https://cloud.digitalocean.com/apps/) and connect it up using GitHub (requires SSO between your GH and DO accounts)
+2. Follow the wizard leaving all the defaults for now - this will produce a failed build. Don't worry
+3. Go to the Settings page on your app and scroll down to see the yaml config. 
+4. Download a copy and augment it and augment from the example files based on your app requirements
```

Comment:
what is this line trying to say?  
download one of the example files and update it based on app requirements?
download the yaml config from settings and augment it with the example files?

Reply:
I'll clean up the english but it probably helps you follow along with DO on another screen so you can get additional context.